lua/telescope/builtin/internal.lua

```diff
@@ -45,6 +45,73 @@ internal.builtin = function(opts)
   }):find()
 end
 
+do
```

Comment:
This do block does nothing.

Reply:
I should have put a comment for the do block. I was preparing to make the function have some state. For example, there would be a table of all user selections, and a stack for previous user selections so going back would just pop and selecting would push. This would allow me to implement going back to the previous menu like in your next comment.